pkg/appmanager/as3Manager.go

```diff
@@ -52,6 +63,42 @@ func (appMgr *Manager) processUserDefinedAS3(template string) bool {
 	return true
 }
 
+// Validates the AS3 Template
+func (appMgr *Manager) validateAS3Template(template string) bool {
+
+	workingDir, _ := os.Getwd()
+	var schemaUrl = workingDir + "/vendor/src/f5/schemas/as3-schema-3.9.0-3-cis.json"
+
+	var schema, err = ioutil.ReadFile(schemaUrl)
+
+	if err != nil {
+		log.Errorf("Schema not found :\n")
+		return false
+	}
+
+	// Load Both the AS3 Schema and AS3 Template
+	schemaLoader := gojsonschema.NewStringLoader(string(schema))
+	documentLoader := gojsonschema.NewStringLoader(template)
+
+	result, err := gojsonschema.Validate(schemaLoader, documentLoader)
+
+	if err != nil {
+		log.Errorf("%s", err)
+		return false
+	}
+
+	if result.Valid() {
+		log.Debugf("AS3 Template is Validated Successfully \n")
+		return true
+	} else {
+		log.Errorf("AS3 Template is not valid. see errors :\n")
+		for _, desc := range result.Errors() {
+			log.Errorf("- %s\n", desc)
+		}
+		return false
```

Comment:
Suggestion: It is a good practice to look for failure block with if condition and put the true block out of `if` check.

```golang
if !result.Valid() {
  log.Errorf("Error")
  return false
}

return true
```